src/components/figures/chart/chartJs/chartjs.ts

```diff
@@ -3,17 +3,32 @@ import { Chart, ChartConfiguration } from "chart.js/auto";
 import { deepCopy } from "../../../../helpers";
 import { Figure, SpreadsheetChildEnv } from "../../../../types";
 import { ChartJSRuntime } from "../../../../types/chart/chart";
-import { chartJsExtensionRegistry, getChartJSConstructor } from "./chart_js_extension";
+import { css } from "../../../helpers";
+import { chartJsExtensionRegistry } from "./chart_js_extension";
 import { chartShowValuesPlugin } from "./chartjs_show_values_plugin";
 import { waterfallLinesPlugin } from "./chartjs_waterfall_plugin";
 
 interface Props {
   figure: Figure;
 }
 
-chartJsExtensionRegistry.add("chartShowValuesPlugin", chartShowValuesPlugin);
-chartJsExtensionRegistry.add("waterfallLinesPlugin", waterfallLinesPlugin);
+css/* scss */ `
+  .o-spreadsheet {
+    .o-chart-custom-tooltip {
```

Comment:
I'm not sure what this css is doing in this commit

Reply:
neither do I ...